cmd/job.go

```diff
@@ -22,6 +22,6 @@ func jobCommand(l log.Logger, conf config.Optimus, pluginRepo models.PluginRepos
 	cmd.AddCommand(jobRenderTemplateCommand(l, conf, pluginRepo))
 	cmd.AddCommand(jobValidateCommand(l, conf, pluginRepo, conf.Project.Name, conf.Host))
 	cmd.AddCommand(jobRunCommand(l, conf, pluginRepo, conf.Project.Name, conf.Host))
```

Comment:
I don't think we use this command, can you remove the code for this command too, it will be little confusing